src/jvm.ts

```diff
@@ -427,6 +427,7 @@ class JVM {
         assert(false, `Invariant failure: Thread pool cannot be emptied post-JVM termination.`);
         return false;
       case JVMStatus.TERMINATING:
+        methods.dumpStats();
```

Comment:
Yup; I will wrap it in a flag. At the very minimum, it should have been under `!RELEASE`.